src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -370,15 +370,17 @@ public class ADDR_Addresses_TDTM extends TDTM_Runnable {
     * @return Date
     */ 
     private static Date dtStartOrEndOfAddrSeasonal(Address__c addr, boolean fStart) {    
+        integer startYear = integer.valueOf(addr.Seasonal_Start_Year__c);
```

Comment:
won't isSeasonalOverlap() need to be modified to look at the years as well (if they are specified)?